### PR TITLE
need advice on building for docker

### PR DESCRIPTION
**Pull request by parallaxisjones**
my changes to the code don't seem to be having any effect when I build my docker container... I'm not sure what's happening.  This PR is just a basic example of what I'm trying to achieve with the changes I've made.  parameterizing the query for a client to owner.

Code needs to be refactored, this was my first forray into making real builable go code... intent is to alter the get endpoint to parse query params and build out the select statement for hydra_client

### CODE REVIEW

**Comment by arekkas** [comment]
How are you trying to build the docker image?

File: Dockerfile [hunk 1]
```diff
@@ -2,15 +2,16 @@ FROM golang:1.8-alpine
...
 RUN go get github.com/Masterminds/glide
+ADD . /go/src/github.com/ory-am/hydra
 WORKDIR /go/src/github.com/ory-am/hydra
 
 ADD ./glide.yaml ./glide.yaml
 ADD ./glide.lock ./glide.lock
 RUN glide install --skip-test -v
 
-ADD . .
+# ADD . .
```
Comment (arekkas) [inline Dockerfile]: Why did you remove this?

File: Dockerfile [hunk 2]
```diff
@@ -2,15 +2,16 @@ FROM golang:1.8-alpine
 
 RUN apk add --no-cache git
 RUN go get github.com/Masterminds/glide
+ADD . /go/src/github.com/ory-am/hydra
```
Comment (arekkas) [inline Dockerfile]: Don't do this, it will force glide install on every code change

File: Dockerfile (hunk 1 above)
Comment (parallaxisjones) [inline Dockerfile]: because I added the following

File: Dockerfile (hunk 2 above)
Comment (parallaxisjones) [inline Dockerfile]: I was trying to force a re-install to get my code to build.

**Comment by parallaxisjones** [comment]
as far as how I'm building the docker container, 
`
go build && go install
docker build -t  oryd/hydra .
`

 then running the container from a docker-compose file with the rest of my services


sorry about the test fail, I had some trouble with the PR because I believe my branch was behind and I had to update and re-merge the changes because I guess there were too many differences to make the pull request from the fork that I made.  I clobbered some of my changes, then pushed it up as I was leaving and didn't check well enough

**Comment by arekkas** [comment]
Do this instead:

```
docker build -t hydra .
```

**Comment by parallaxisjones** [comment]
thanks.  That was totally it, I wasn't tagging the image correctly, I was using latest-http in my docker compose file. 

I got my owners query working, but it's not code good enough to offer a PR yet and it's not safe sql. I have a conference demo thing coming up in 2 weeks, but after that I'll be able to clean up my code & make tests and offer an official PR.  I would love to be  a hydra contributor

**Comment by arekkas** [comment]
> I got my owners query working, but it's not code good enough to offer a PR yet and it's not safe sql. I have a conference demo thing coming up in 2 weeks, but after that I'll be able to clean up my code & make tests and offer an official PR. I would love to be a hydra contributor

Great! Thank you for your hard work on this and please excuse my late reply. Welcome to the Hydra family :)

**Comment by arekkas** [comment]
closing due to inactivity, feel free to reopen
